include/transformationTuffer_kopie.h

```diff
@@ -0,0 +1,96 @@
+#ifndef TRANSFORMATIONBUFFER_H
+#define TRANSFORMATIONBUFFER_H
+
+#include "typedefs.h"
+#include "stampedTransformation.h"
+#include <list>
+
+/**
+ * @brief The TransformationBuffer class
+ */
+class TransformationBuffer
+{
+
+friend class GMLWriter;
+
+public:
+
+    /**
+     * @brief TransformationBuffer
+     * @param d
+     */
+    TransformationBuffer(const DurationSec &d)
+    : storageTime(d)
+    {}
+
+    /**
+     * @brief oldestEntry
+     * @param te
+     */
+    void oldestEntry(StampedTransformation &te);
```

Comment:
This function should be `const` if it does not modify the `TransformationBuffer` object.